package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -245,12 +246,13 @@ class DistanceMatrix(AnalysisBase):
          :class:`MDAnalysis.analysis.base.Results` instance.
 
     """
-    def __init__(self, universe, select='all', metric=rmsd, cutoff=1E0-5,
+    def __init__(self, atomGroup, select='all', metric=rmsd, cutoff=1E0-5,
```

Comment:
This is missing the point of the change. Currently a Universe and selection string are both passed to construct an AtomGroup. The desired change is for this selection to happen outside of the function, so these two arguments are replaced by just an AtomGroup. To not break things abruptly, this will therefore require for a while accepting either U+selection (and warning about the change) OR AG.

Reply:
I see. I thought we'd have AG along with `selection` anyways should the need arise. I'll make the necessary changes to accept AG (without `selection`) along with `Universe` and `selection`.